### PR TITLE
Outlier detection

### PR DESCRIPTION
**Pull request by andrewhead**
See scholar repo issue https://github.com/allenai/scholar/issues/26282

This PR includes three changes to support filtering out bounding boxes with overly large bounding boxes:

1. It detects very large bounding boxes for symbols in the pipeline scripts (`upload.py`)
2. It adds a field called `tags` to all entities that can contain arbitrary tags (in this case to support the `"large"` tag for entities with very large bounding boxes)
3. It excludes all entities with the "large" tag from being rendered in the UI. 

The detection of large bounding boxes in the `upload.py` script is an alternative to doing so with the SQL maintenance script that was first proposed in https://github.com/allenai/scholarphi-pipeline/pull/27.

### CODE REVIEW

File: ui/src/components/entity/EntityAnnotationLayer.tsx
Comment (mjlangan) [reply ui/src/components/entity/EntityAnnotationLayer.tsx:204]: though I will also walk it back in that it might be useful to do something for debugging if/when we hit this situation.

**Review by mjlangan — APPROVED** [review]
Pending the tests passing, LGTM. We could possibly log something to the console when filtering out a bounding box, but I'll leave that up to your discretion.

**Comment by andrewhead** [comment]
> We could possibly log something to the console when filtering out a bounding box, but I'll leave that up to your discretion.

Good thinking! I have added a console log statement for this case.

File: ui/src/components/entity/EntityAnnotationLayer.tsx
Comment (andrewhead) [reply ui/src/components/entity/EntityAnnotationLayer.tsx:204]: I think you and Alex make a good point about this. I agree that it would be useful to be able to visualize large bounding boxes in the user interface, without having to make changes in the API. So let us start out with this code in the user interface, and we will migrate it to the API when we are feeling that we have done all that we want to do for filtering out bounding boxes.

**Pull request merged**
